vj4/handler/contest.py

```diff
@@ -85,6 +85,23 @@ class ContestCommonOperationMixin(object):
                                                        ranked_tsdocs, udict, pdict)
     return tdoc, rows
 
+  async def convert_and_verify_pids_str(self, pids: str):
```

Comment:
`_str` is ugly...

Reply:
could you suggest a better name?

Reply:
No, could you suggest a better name?

Reply:
Obviously if I have had a better name, it should be in the commit. `_str` is find, since it emphasizes that its receiving a pids string, instead of a list generally.

Reply:
It should be `convert_and_verify_pids(self, pids: list)`, where `pids=list(set(map(...)))`

Reply:
Methods to parse pids should be extracted as well

Reply:
So there should be 2 functions.